lib/travis/build/addons/deploy/script.rb

```diff
@@ -40,18 +40,18 @@ def check_conditions_and_run
               end
 
               sh.else do
-                failure_message_unless(repo_condition, "this repo's name does not match one specified in .travis.yml's deploy.on.repo: #{on[:repo]}")
-                failure_message_unless(branch_condition, "this branch '$TRAVIS_BRANCH' is not permitted to deploy")
-                failure_message_unless(runtime_conditions, "this is not on the required runtime")
-                failure_message_unless(custom_conditions, "a custom condition was not met")
-                failure_message_unless(tags_condition, "this is not a tagged commit")
+                warning_message_unless(repo_condition, "this repo's name does not match one specified in .travis.yml's deploy.on.repo: #{on[:repo]}")
+                warning_message_unless(branch_condition, "this branch does not meet a condition")
```

Comment:
Okay I can see that, and am happy not to adjust that part much. How about simply "this branch is not permitted"?